src/integrations/GA4/browser.js

```diff
@@ -67,6 +67,11 @@ export default class GA4 {
       window.gtag('config', measurementId, gtagParameterObject);
     }
 
+    // If userTraits available, setting it as a part of global gtag object
+    if (this.analytics.userTraits) {
+      window.gtag('set', 'user_properties', flattenJsonPayload(this.analytics.userTraits));
+    }
+
```

Comment:
Also in identify call lets do same in case without page refresh someone wants to update the user properties

Reply:
we are already updating user_properties as a part of identify call

Reply:
I see it's already set - https://github.com/rudderlabs/rudder-sdk-js/pull/1148/files#diff-5d49e897e76914916677a00e9dc67256d66a6c692687aa1dc07c3dbd2763842aR231
Anything else you are implying ?

Reply:
No I missed it